internal/characters/candace/asc.go

```diff
@@ -0,0 +1,29 @@
+package candace
+
+import (
+	"math"
+
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/player/character"
+	"github.com/genshinsim/gcsim/pkg/modifier"
+)
+
+const a4Key = "candace-a4"
+
+func (c *char) a4(char *character.CharWrapper, duration int) {
+	char.AddAttackMod(character.AttackMod{ // TODO: is this right implementation?
```

Comment:
I assume we should be using `AttackMod` here for the damage buff, but needs verification.